src/OrchardCore.Modules/OrchardCore.Google/TagManager/Recipes/GoogleTagManagerSettingsStep.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Threading.Tasks;
+using OrchardCore.Entities;
+using OrchardCore.Google.TagManager.Settings;
+using OrchardCore.Google.TagManager.ViewModels;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Google.TagManager.Recipes
+{
+    /// <summary>
+    /// This recipe step sets Google TagManager settings.
+    /// </summary>
+    public class GoogleTagManagerSettingsStep : IRecipeStepHandler
```

Comment:
We can drop this now too :) it gets handled by the site settings recipe step handler